backend/backend/public_urls.py

```diff
@@ -22,6 +22,7 @@
 
 path_prefix = settings.PATH_PREFIX
 api_path_prefix = settings.API_DEPLOYMENT_PATH_PREFIX
+simple_prompt_studio_path_prefix = settings.SIMPLE_PROMPT_STUDIO_PATH_PREFIX
```

Comment:
@tahierhussain Let us move this line into the try-catch block. It may cause OSS itself to fail if the config is missing.

Reply:
Thanks for the catch, @harini-venkataraman. Done.